spec/services/comment_service_spec.rb

```diff
@@ -8,10 +8,12 @@
 
   describe 'create comment' do
     context 'when success' do
+      let(:count_comment_after_create) { 1 }
       subject(:service) { described_class.new(comments_form_params) }
 
       it do
-        # expect(service.save_comment).to eq(true)
+        service.save_comment
```

Comment:
`expect { service.save_comment }.to change(Comment, :count).from(0).to(count_comment_after_create)`
Please do something like this in another similar specs